app/controllers/admin/dashboards_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::DashboardsController < Admin::ApplicationController
```

Comment:
__[rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->